packages/react-components/src/video-player/videoPlayer.tsx

```diff
@@ -449,12 +452,12 @@ export class VideoPlayer extends React.Component<IVideoPlayerProps, IVideoPlayer
     }
   };
 
-  private errorHandlingForLiveMode = (errorMessage: string) => {
+  private errorHandlingForLiveMode(errorMessage: string) {
```

Comment:
this change may cause `this` to be lost, better to keep the arrow function to be safe

Reply:
> this change may cause `this` to be lost, better to keep the arrow function to be safe

yes good catch, i was using this to trace call stack. forget to remove it. Thanks! Removed.